src/main.jsx

```diff
@@ -3,8 +3,4 @@ import ReactDOM from "react-dom/client";
 import App from "./App";
 import "./index.css";
 
-ReactDOM.createRoot(document.getElementById("root")).render(
-  <React.StrictMode>
```

Comment:
Any particular reason for removing this line?

(things like this can be described in the PR description. Can be quite helpful for reviewers)

Reply:
Sorry Sir @nsptel, Here I am attaching a link for you
https://stackoverflow.com/a/61897567/17791486